test/Makefile.defs

```diff
@@ -361,8 +361,12 @@ endif
 # Header include path + linker flag for libomptest based OMPT tests
 OMPTEST = -I$(AOMP)/lib/omptest/include -lomptest
 
+GPU_MEM_TOTAL_SIZE = $(shell $(AOMPHIP)/bin/rocm-smi --showmeminfo vram | grep -E "VRAM Total Memory"  | grep -m1 -Eo " [0-9]+" | tr -d ' ')
+GPU_MEM_TOTAL_SIZE_GIB := $(shell bc -l <<< "$(GPU_MEM_TOTAL_SIZE)/1024 ^ 3")
+GPU_MEM_AVAIL_SIZE = $(shell $(AOMPHIP)/bin/rocm-smi --showmeminfo vram | grep -E "VRAM Total Used Memory"  | grep -m1 -Eo " [0-9]+" | tr -d ' ')
+GPU_MEM_AVAIL_SIZE_GIB := $(shell bc -l <<< "$(GPU_MEM_AVAIL_SIZE)/1024 ^ 3")
```

Comment:
I think  `GPU_MEM_AVAIL_SIZE` should be `GPU_MEM_USED_SIZE` for clarity. And then:
`GPU_MEM_AVAIL_SIZE = GPU_MEM_TOTAL_SIZE - GPU_MEM_USED_SIZE`
`GPU_MEM_SIZE_GIB = GPU_MEM_AVAIL_SIZE/1024 ^ 3`

That way we only have one conversion to GiB.